libmamba/src/core/download.cpp

```diff
@@ -10,17 +13,88 @@
 namespace mamba
 {
 
+    namespace
+    {
+        void init_remote_fetch_params(Context::RemoteFetchParams& remote_fetch_params)
+        {
+            if (!remote_fetch_params.curl_initialized)
+            {
+                if (remote_fetch_params.ssl_verify == "<false>")
+                {
+                    LOG_DEBUG << "'ssl_verify' not activated, skipping cURL SSL init";
+                    remote_fetch_params.curl_initialized = true;
+                    return;
+                }
+#ifdef LIBMAMBA_STATIC_DEPS
+                CURLHandle handle;
+                auto init_res = handle.get_ssl_backend_info();
+                switch (init_res.second)
+                {
+                    case CurlLogLevel::kInfo:
```

Comment:
Nit: remove C-style `k` prefix from enums?

Reply:
IIRC there were some errors due to conflict with `windows` or `osx` keywords.